velox/connectors/hive/HiveConnectorUtil.cpp

```diff
@@ -847,4 +854,143 @@ core::TypedExprPtr extractFiltersFromRemainingFilter(
   return expr;
 }
 
+namespace {
+
+#ifdef VELOX_ENABLE_PARQUET
+std::optional<TimestampUnit> getTimestampUnit(
```

Comment:
`getTimestampUnit` and `getTimestampTimeZone` is not parquet specific

Reply:
@Yuhta Thanks for your catch. It should be `VELOX_ENABLE_ARROW`. I have updated. Can you help to review again? Thanks.

Reply:
@Yuhta It seems we still need to use VELOX_ENABLE_PARQUET here. Otherwise the build will failed with following exceptions:

`error: ‘getTimestampUnit’ was not declared in this scope; did you mean ‘TimestampUnit’?`